tests/integration-tests/tests/performance_tests/test_startup_time/test_startup_time/pcluster.config.yaml

```diff
@@ -0,0 +1,57 @@
+Region: {{ region }}
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: c5n.xlarge
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    AdditionalIamPolicies:
+      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore #Required to report patching status
+      - Policy: arn:aws:iam::aws:policy/CloudWatchFullAccess # Required to add performance testing widgets to the cluster dashboard
```

Comment:
Do we expect the head node to create the cloudwatch widget?
Is the DevSetting `ComputeStartupTimeMetricEnabled` that makes the head node create the widget?